example.env

```diff
@@ -0,0 +1,4 @@
+DISCORD_TOKEN="Token which you get from https://discord.com/developers/applications/"
+MANAGER_ID="discord ID of the person managing the bot"
```

Comment:
Will this be loaded as an int? Otherwise, this will break the manager thing, and nobody will be able to kill the bot in a stable way. Other than this everything looks good

Reply:
I made sure to load it as an int in `thenewboston.py` file.